Source/FolioReaderPage.swift

```diff
@@ -157,7 +157,7 @@ public class FolioReaderPage: UICollectionViewCell, UIWebViewDelegate, UIGesture
         }
         
         UIView.animateWithDuration(0.2, animations: {webView.alpha = 1}) { finished in
-            webView.isColors = false
+            (webView as? FolioReaderWebView)?.isColors = false
```

Comment:
Lets reduce the `(webView as? FolioReaderWebView)` cast, we can just cast with guard on beginning of the function.

```swift
guard let webView = webView as? FolioReaderWebView else { return }
```

Then just use `webView` as previously.